service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/MessageJudgeControllerSubmittedTest.java

```diff
@@ -69,7 +71,7 @@ void shouldNotifyJudicialMessageRecipientWhenNewJudicialMessageAdded() throws No
 
         Map<String, Object> expectedData = Map.of(
             "respondentLastName", "Davidson",
-            "caseUrl", "http://fake-url/cases/case-details/12345#JudicialMessagesTab",
+            "caseUrl", buildCaseUrl(JUDICIAL_MESSAGES, CASE_REFERENCE),
```

Comment:
not sure I am fan of this, it reduces the code readability of the expected data. Now if someone was to look at these tests they would have to go and look and see what this method does to be able to work out what the expected data is.